core/src/main/java/de/unistuttgart/iaas/amyassist/amy/core/speech/TextToSpeech.java

```diff
@@ -108,14 +109,24 @@ private void speak(LineListener listener, String s) {
 	 * Voice Output
 	 * 
 	 * @param listener
-	 *            TODO
+	 *            Listener for the Output-Clip
 	 * @param s
 	 *            String to say
 	 */
 	public void say(LineListener listener, String s) {
 		speak(listener, preProcessing(s));
 	}
 
+	/**
+	 * Logs the Output String
+	 * 
+	 * @param s
+	 *            String to Log
+	 */
+	public void log(String s) {
```

Comment:
log should not be public

Reply:
wird aufgerufen, wenn Ein der Recognizer von z.B. Arbeit aus aufgerufen wird, via Stream des Audios. Damit Amy nicht einfach anfängt zu reden, ich kann die Methode anders nennen, sie gibt den Output einfach nur als Log aus, anstatt als Log und Audio.

Reply:
mach mal aus TTS ein Interface